panel/src/components/Forms/Field/StructureField.vue

```diff
@@ -563,6 +564,13 @@ export default {
 			} else {
 				return true;
 			}
+		},
+		/**
+		 * When the field's value changes
+		 * @param {array} values
```

Comment:
I'm not sure but is this docblock correct?

Reply:
It was ambiguous and duplicated from an existing method…
Fixed.